lib/modules/platform/github/index.ts

```diff
@@ -1378,6 +1379,55 @@ export async function ensureIssueClosing(title: string): Promise<void> {
   }
 }
 
+async function addMilestone(
+  issueNo: number,
+  milestoneTitle: string,
+): Promise<void> {
+  logger.debug(`Adding milestone '${milestoneTitle}' to #${issueNo}`);
+  // first, find the milestone id
+  const milestones = await getMilestones(issueNo);
+  const milestone = milestones.find((m) => m.title === milestoneTitle);
+  if (!milestone) {
+    logger.warn(
+      `Milestone '${milestoneTitle}' did not exists. Adding of milestone skipped.`,
+    );
+    return;
+  }
+
+  // then, add the milestone id to the PR
+  const repository = config.parentRepo ?? config.repository;
+  await githubApi.patchJson(`repos/${repository}/issues/${issueNo}`, {
```

Comment:
It's definitely the issues API and not pulls API?

Reply:
Yes, [the docs](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#about-pull-requests) say:

> Pull requests are a type of issue. Any actions that are available in both pull requests and issues, like managing assignees, labels, and milestones, are provided by the REST API to manage issues. For more information, see "[Issues](https://docs.github.com/en/rest/issues)."